static/form.js

```diff
@@ -306,8 +306,24 @@ function checkInputRadius(inputId, errorMsg){
   const inputElement = document.getElementById(inputId);
   const errorMessage = document.getElementById(errorMsg);
 
+  // Check if the input are all digits.
+  const regex = /^\d+$/;
+  let areDigits = regex.test(inputElement.value);
+
+  if(!areDigits){
+    errorMessage.textContent = 'Please enter a valid radius between 0 and 100 (Integers).';
+    return false;
+  }
+
   // Get the entered value and convert it to an integer
-  const inputValue = parseInt(inputElement.value, 10);
+  let inputValue = parseFloat(inputElement.value, 10);
```

Comment:
can `inputElement` and `inputValue` please be renamed to something better?  yes, they are inputs from the user, but so is the path width information.  the code to compare `inputValue` and `pathWidthVal` below would make so much more sense it if was using, say, `radius` and `pathWidth` as the variables it compares and computes with, no?